channels/jaicp/src/main/kotlin/com/justai/jaicf/channel/jaicp/reactions/JaicpReactions.kt

```diff
@@ -20,11 +24,40 @@ open class JaicpReactions : Reactions() {
         return SayReaction.create(text)
     }
 
-    fun collect(): JsonObject {
-        val jsonReplies = replies.map { reply ->
-            reply.serialized().toJson()
-        }
+    /**
+     * Starts a new conversation session with client.
+     *
+     * Each session is logged to JAICP analytics with unique session identifier.
+     * Using this reaction creates new session identifier, clears existing [DialogContext]
+     *  and sends a response with new session identifier.
+     *
+     * @see [com.justai.jaicf.channel.jaicp.logging.JaicpConversationLogger]
+     * */
+    fun startNewSession() {
```

Comment:
The documentation says this function relates to the logging feature. Thus it maybe a bit more correct to rename it somehow or create an inner object like `logging` (or `conversation`) to have `reactions.jaicp?.logging?.startSession()`

Reply:
I don't think so. Session management also relates to `session` in `BotContext`. We cannot separate it to represent only logging semantics, otherwise it will be unclear and misleading.